.ci/pipeline_definitions

```diff
@@ -43,9 +31,3 @@ gardener-website-generator:
                 - dimitar.kostadinov@sap.com
                 - jordan.jordanov@sap.com
                 - nikolay.boshnakov@sap.com
-    # update-deps:
-    #   traits:
-    #     update_component_deps:
-    #       merge_policy: auto_merge
-    #     cronjob:
-    #       interval: '15m'
```

Comment:
comments are of course safe to omit, you are using git for versioning anyways ;-)